students/ejackie/session04/trigrams_dict.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+##################################################
+## Author: Jackie Cheung
+## Date: 2018/10/30
+## Version: 0.1
+## Class: Au2018-Py210B
+## Description: Trigrams – Simple Text Manipulation (Kata Fourteen: Tom Swift Under Milk Wood)
+##################################################
+
+import sys
+import random
+
+#words = "I wish I may I wish I might".split()
+
+def read_in_data(filename):
+    """
+    read text from specified file
+
+    """ 
+    with open(filename, 'r') as f:
+        in_data = f.read()
+    f.closed
+    True
```

Comment:
Minor: I suppose these lines 22-23 were your debugging purpose only? No need for these lines otherwise. Just FYI.